gcp/appengine/handlers.py

```diff
@@ -59,6 +59,7 @@ def make_affected_commits_public(bug):
   """Make related AffectedCommit entities public."""
```

Comment:
Did a quick search and found this: https://github.com/google/osv.dev/blob/0e31a369fb9055dab2ddf97e0ec3ed7ef5978823/docker/worker/worker.py#L189-L200

Does this also need some handling for the new `AffectedCommits`?

Reply:
Good catch. I've fixed this up as well.